navigator/backend/src/test/scala/com/digitalasset/navigator/backend/IntegrationTest.scala

```diff
@@ -35,12 +32,19 @@ import scala.concurrent.duration._
 
 class IntegrationTest
     extends AsyncFreeSpec
-    with SandboxFixture
-    with AkkaBeforeAndAfterAll
+    with CantonFixture
     with SuiteResourceManagementAroundAll
     with Matchers {
   self: Suite =>
 
+  override protected def authSecret = None
+  override protected def darFiles = List()
+  override protected def devMode = false
+  override protected def nParticipants = 1
+  override protected def timeProviderType = TimeProviderType.Static
```

Comment:
Does the test actually require Static time ?

Reply:
It doesn't, is the preference to not use static where possible?